server/api/event/register/index.post.ts

```diff
@@ -71,6 +71,23 @@ export default defineEventHandler(async (event) => {
       })
     })
 
+  // check if user meets registration requirements
+  const meetsProgrammeRequirement =
+    data[eventUID]?.registration?.requirements?.programmes?.includes(
+      user?.studyProgram
+    ) ?? true
+  const meetsYearRequirement =
+    data[eventUID]?.registration?.requirements?.years?.includes(user?.year) ??
```

Comment:
The year key is called `currentYear`, not `year`. That is why it is not working.